config/install/format_strawberryfield.settings.yml

```diff
@@ -0,0 +1,3 @@
+iiif_settings:
```

Comment:
I would rather set this here, but I don't know how to test if it's working without reinstalling? So it is still also set in the IiifSettings form hardcoded on lines 76 and 84.